test/java/org/opendatakit/briefcase/pull/aggregate/AggregateAttachmentTest.java

```diff
@@ -1,19 +1,39 @@
 package org.opendatakit.briefcase.pull.aggregate;
 
-import static org.junit.Assert.assertThat;
-
 import java.io.IOException;
+import java.net.MalformedURLException;
 import java.net.URISyntaxException;
+import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import org.hamcrest.Matchers;
 import org.junit.Test;
 
+import static org.junit.Assert.*;
+
 public class AggregateAttachmentTest {
   @Test
   public void computes_the_string_MD5_hash_of_a_file() throws URISyntaxException, IOException {
     Path file = Paths.get(AggregateAttachmentTest.class.getResource("/org/opendatakit/briefcase/pull/aggregate/lorem-ipsum-40k.txt").toURI());
     String expectedHash = "E79C0D4AD451003BA8CFDC1183AC89E9";
     assertThat(AggregateAttachment.md5(file), Matchers.is(expectedHash));
   }
+
+  @Test
+  public void gets_download_url() throws URISyntaxException, MalformedURLException {
```

Comment:
```suggestion
  public void gets_download_url() throws MalformedURLException {
```
`UriSyntaxException` is not being thrown by this block